tests/epyccel/recognised_functions/test_math_funcs.py

```diff
@@ -377,12 +382,735 @@ def floor_return_type_real(x):
         a = floor(x)
         return a
 
-    f1 = epyccel(floor_return_type_int)
+    f1 = epyccel(floor_return_type_int, language = language)
     x = randint(100)
     assert(isclose(f1(x) ,  floor_return_type_int(x), rtol=1e-15, atol=1e-15))
     assert(type(f1(x))  == type(floor_return_type_int(x))) # pylint: disable=unidiomatic-typecheck
 
-    f1 = epyccel(floor_return_type_real)
+    f1 = epyccel(floor_return_type_real, language = language)
     x = randint(100)
     assert(isclose(f1(x) ,  floor_return_type_real(x), rtol=1e-15, atol=1e-15))
     assert(type(f1(x))  == type(floor_return_type_real(x))) # pylint: disable=unidiomatic-typecheck
+
+#------------------------------- Ceil function -------------------------------#
+def test_ceil_call(language):
+    @types('real')
+    def ceil_call(x):
+        from math import ceil
+        return ceil(x)
+
+    f1 = epyccel(ceil_call, language = language)
+    x = rand()
+    assert(ceil_call(x) == f1(x))
+
+    assert isinstance(ceil_call(x), type(f1(x)))
```

Comment:
I think it would be good to add an integer test here. It will ensure that `ceil` isn't implemented naively  (e.g. `ceil = lambda x: int(x)+1`)

Reply:
done